_doc-build-windows/action.yml

```diff
@@ -189,6 +223,12 @@ runs:
       shell: powershell
       run: python -m pip install -U pip
 
+    - name: "Install Jupyter for Quarto"
+      if: ${{ inputs.needs-quarto == 'true' }}
+      shell: bash
+      run: |
+        python -m pip install jupyter nbformat
```

Comment:
Is this required for Quarto to operate? This may be in conflict with packages declared inside of the libraries triggering the actions.

Reply:
We need jupyter to convert `qmd` files to pdf format

Reply:
As discussed, projects should be the ones adding their doc-extensions in the required extra target.

Reply:
sure added it, i will also add this requirement as a note in ansys sphinx theme docs